tests/test_agent_langchain.sh

```diff
@@ -69,38 +94,69 @@ function validate() {
 
 function validate_microservice() {
     echo "Testing agent service"
-    local CONTENT=$(curl http://${ip_address}:5042/v1/chat/completions -X POST -H "Content-Type: application/json" -d '{
+    local CONTENT=$(http_proxy="" curl http://${ip_address}:9090/v1/chat/completions -X POST -H "Content-Type: application/json" -d '{
      "query": "What is Intel OPEA project?"
     }' | tee ${LOG_PATH}/test-agent-langchain.log)
     local EXIT_CODE=$(validate "$CONTENT" "OPEA" "test-agent-langchain")
     echo "$EXIT_CODE"
     local EXIT_CODE="${EXIT_CODE:0-1}"
     echo "return value is $EXIT_CODE"
     if [ "$EXIT_CODE" == "1" ]; then
-        docker logs test-comps-tgi-gaudi-service &> ${LOG_PATH}/test-comps-tgi-gaudi-service.log
-        docker logs test-comps-langchain-agent-endpoint &> ${LOG_PATH}/test-comps-langchain-agent-endpoint.log
+        echo "==============tgi container log ==================="
+        docker logs test-comps-tgi-gaudi-service
+        echo "==============agent container log ==================="
+        docker logs comps-agent-endpoint
         exit 1
     fi
 }
 
-function stop_docker() {
+function stop_tgi_docker() {
     cid=$(docker ps -aq --filter "name=test-comps-tgi-gaudi-service")
     echo "Stopping the docker containers "${cid}
     if [[ ! -z "$cid" ]]; then docker rm $cid -f && sleep 1s; fi
-    cid=$(docker ps -aq --filter "name=test-comps-langchain-agent-endpoint")
+    echo "Docker containers stopped successfully"
+}
+
+function stop_agent_docker() {
+    cid=$(docker ps -aq --filter "name=comps-agent-endpoint")
     echo "Stopping the docker containers "${cid}
     if [[ ! -z "$cid" ]]; then docker rm $cid -f && sleep 1s; fi
     echo "Docker containers stopped successfully"
 }
 
+function stop_docker() {
+    stop_tgi_docker
+    stop_agent_docker
+}
+
 function main() {
 
     stop_docker
 
     build_docker_images
-    start_service
 
+    start_tgi_service
+
+    # test react_langchain
+    start_react_langchain_agent_service
+    echo "=============Testing ReAct Langchain============="
+    validate_microservice
+    stop_agent_docker
+    echo "============================================="
+
+    # # test react_langgraph
```

Comment:
why comment those test?

Reply:
This test needs to use OpenAI model GPT-4o-mini. @chensuyue